src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
It's better to reset margins and paddings only for some specified selectors with default styles.